src/components/Sweep.tsx

```diff
@@ -0,0 +1,221 @@
+                <AssetIcon size='xs' src={asset.icon} />
+              </Flex>
+              <Box position='relative'>
+                <Box
+                  position='absolute'
+                  width='50px'
+                  height='auto'
+                  borderBottomWidth={2}
+                  borderStyle='dotted'
+                  borderColor='border.base'
+                  transform='rotate(90deg)'
+                />
+                <Box
+                  position='absolute'
+                  width='50px'
+                  height='auto'
+                  borderBottomWidth={2}
+                  borderStyle='dotted'
+                  borderColor='border.base'
+                  transformOrigin='0 0'
+                  transform='rotate(-135deg) translate(-45%)'
+                />
+                <Box
+                  position='absolute'
+                  width='50px'
+                  height='auto'
+                  borderBottomWidth={2}
+                  borderStyle='dotted'
+                  borderColor='border.base'
+                  transformOrigin='100% 0'
+                  transform='rotate(-45deg) translate(30%)'
+                />
+                <AssetIcon size='md' src={asset.icon} />
+              </Box>
+            </Flex>
+          </Stack>
+          <Stack>
+            <CText color='text.subtle'>
+              {translate('modals.send.consolidate.body', { asset: asset.name })}
+            </CText>
+          </Stack>
+          <Stack justifyContent='space-between'>
+            <Button
+              onClick={handleSweep}
+              disabled={isEstimatedFeesDataLoading || isSweepPending}
+              size='lg'
+              colorScheme={'blue'}
+              width='full'
+              data-test='utxo-sweep-button'
+              isLoading={isEstimatedFeesDataLoading || isSweepPending}
+              loadingText={translate('common.loadingText')}
+            >
+              {translate('modals.send.consolidate.consolidateFunds')}
+            </Button>
+            <Button
+              onClick={handleBack}
+              size='lg'
+              width='full'
+              colorScheme='gray'
+              isDisabled={false}
+            >
+              {translate('common.cancel')}
+            </Button>
+          </Stack>
+        </Stack>
+        <Stack px={2}>
+          <Row>
+            <Row.Label>{translate('modals.approve.estimatedGas')}</Row.Label>
+            <Row.Value>
+              <Skeleton isLoaded={isEstimatedFeesDataSuccess}>
+                <Box textAlign='right'>
+                  <Amount.Fiat value={estimatedFeesData?.txFeeFiat ?? '0'} />
+                  <Amount.Crypto
+                    color='text.subtle'
+                    value={fromBaseUnit(
+                      estimatedFeesData?.txFeeCryptoBaseUnit ?? '0',
+                      asset.precision,
+                    ).toString()}
```

Comment:
already returned as a string
```suggestion
                    )}
```

Reply:
https://github.com/shapeshift/web/pull/5614/commits/e8b8dbdb48c45cb5271046383e86c71d212b9544